docs/QSG/docker.md

```diff
@@ -1,212 +1,95 @@
-- `DAOS_REPOS_NOAUTH`: Space separated list of repos to use without GPG authentication
-	(default "")
-
-For example, building a DAOS base image, with authentication disabled, could be done with the
-following command:
-
-```bash
-$ docker build --tag daos-base:rocky8.4 --build-arg DAOS_AUTH=no \
-	https://github.com/daos-stack/daos.git#master:utils/docker/vcluster/daos-base/el8
-```
-
-It is also possible to build the `daos-base` image from a local tree with the following command:
-
-```bash
-$ docker build --tag daos-base:rocky8.4 utils/docker/vcluster/daos-base/el8
-```
-
-### DAOS Nodes Images
-
 The three images `daos-server`, `daos-admin` and `daos-client` could be built directly from GitHub
-or from a local tree in the same way as for the `daos-base` image.  Following command could be used
-to build directly the three images from GitHub:
+or from a local tree in the same way as for the `daos-base` image.
 
-```bash
-$ for image in daos-server daos-admin daos-client ; do \
-	docker build --tag "$image:rocky8.4" \
-		"https://github.com/daos-stack/daos.git#master:utils/docker/vcluster/$image/el8"; \
-  done
+First you will need to install dockerand docker compose, for eaxample
 ```
+sudo apt install docker
+sudo apt install docker-compose
+....
 
-The Docker file of the `daos-server` image accept the following arguments:
-
-- `DAOS_BASE_IMAGE`: Base docker image to use (default "daos-base")
-- `DAOS_BASE_VERSION`: Version of the base docker image to use (default "rocky8.4")
-- `DAOS_AUTH`: Enable DAOS authentication when set to "yes" (default "yes")
-- `DAOS_HUGEPAGES_NBR`: Number of huge pages to allocate for SPDK (default 4096)
-- `DAOS_SCM_SIZE`: Size in GB of the RAM emulating SCM devices (default 4)
-- `DAOS_BDEV_SIZE`: Size in GB of the file created to emulate NVMe devices (default 16)
-- `DAOS_IFACE_NAME`: Fabric network interface used by the DAOS engine (default "eth0")
-
-!!! note
-    The IP address of the network interface referenced by the `DAOS_IFACE_NAME` argument will be
-    required when starting DAOS.
-
-The Dockerfile of the `daos-client` and `daos-admin` images accept the following arguments:
-
-- `DAOS_BASE_IMAGE`: Base docker image to use (default "daos-base")
-- `DAOS_BASE_VERSION`: Version of the base docker image to use (default "rocky8.4")
-- `DAOS_AUTH`: Enable DAOS authentication when set to "yes" (default "yes")
-- `DAOS_ADMIN_USER`: Name or uid of the daos administrattor user (default "root")
-- `DAOS_ADMIN_GROUP`: Name or gid of the daos administrattor group (default "root")
-
-From a local tree, a more straightforward way to build these images could be done with
-`docker-compose` and the following commands:
+Following command could be used to build directly the three images from GitHub:
 
 ```bash
-$ docker-compose --file utils/docker/vcluster/docker-compose.yml -- build
+docker build --tag "$image:rocky8.4" \ "https://github.com/daos-stack/daos.git#master:utils/docker/vcluster/$image/el8"
```

Comment:
Here, the docker command line is not correct. The command line (at line 157 of the original file) using docker-compose command should be used.
With this last command, the argument should be passed through the .env file (as indicated at lines 163-165).